src/CMakeLists.txt

```diff
@@ -21,8 +21,13 @@ elseif(ARCH_DIR STREQUAL "x86_64" OR ARCH_DIR STREQUAL "amd64"  # Windows detect
         OR CMAKE_OSX_ARCHITECTURES STREQUAL "x86_64")
     set(ARCH_DIR intel64)
 endif()
-if(GENERATOR_IS_MULTI_CONFIG_VAR)
-    set(ARCH_DIR ${ARCH_DIR}/${CMAKE_BUILD_TYPE})
+
+if(WIN32 OR APPLE)
```

Comment:
I suppose we need to drop `WIN32 OR APPLE` as multi-config generator logic does not depend on platform.

Reply:
We shouldn't add Release/Debug suffix on Linux